README.md

```diff
@@ -14,7 +14,22 @@ imports immediately. But this becomes tedious, especially if you use explicit im
 
 _Importify_ can remove unused imports automatically.
 
-In the future, we plan for _Importify_ to be able to add used symbols to imports automatically, so that you won't have to manage imports manually at all. To speed up _Importify_ runs, we will implement a cache server that caches each _Hackage_ package. You will be able to upload caches for your projects there as well. Also on the roadmap is functionality to convert imports between _implicit_ and _explicit_, and between _qualified_ and _unqualified_ forms. Finally, we will be adding functionality to resolve merge conflicts in import section automatically.
+In the future, we plan for _Importify_ to be able to:
+
+ + Add missing imports automatically, so you won't have to manage
+   imports manually at all.
+ + Implement a cache server with the following features:
+   + Download caches for Hackage packages to speed up _Importify_ runs.
+   + Upload your caches for yet-to-be-published FOSS projects to
+     make it easier to collaborate.
+   + Query mappings from any module of every package to symbols
+     exported by it to write your refactoring tools.
+ + Convert imports between _implicit_ and _explicit_, and between
+   _qualified_ and _unqualified_ forms.
+ + Add symbols that are not present in the import section of a module.
```

Comment:
> Add missing imports

and

> Add symbols that are not present in the import section of a module

basically are the same. So you should instead just write only `Add missing imports` or write `And missing imports and symbols that are not present in the import section` inside one list item.